extensions/sql-migration/src/constants/strings.ts

```diff
@@ -1784,7 +1784,7 @@ export function MISSING_TARGET_TABLES_COUNT(missingCount: number): string {
 export function UNAVAILABLE_SOURCE_TABLES_COUNT(unavailableCount: number): string {
 	return localize('sql.migration.table.unavailable.count', "Unavailable for data migration ({0})", formatNumber(unavailableCount));
 }
-export const MISSING_TABLES_HEADING = localize('sql.migration.missing.tables.heading', 'All of tables below are missing on target. To migrate the data in these tables select the migrate schema option above.');
+export const MISSING_TABLES_HEADING = localize('sql.migration.missing.tables.heading', 'All of tables below are missing on target. To migrate the data in these tables, select the migrate schema option above. If the migration schema option is selected and no tables are selected, migrate schema only.');
```

Comment:
this part seems uclear

Reply:
Rework and share the screen shot in teams channel.